examples/test_helper.rs

```diff
@@ -31,10 +33,20 @@ fn test_helper_err(ctx: &Context, args: Vec<RedisString>) -> RedisResult {
     Ok(().into())
 }
 
-fn add_info(ctx: &InfoContext, _for_crash_report: bool) {
-    if ctx.add_info_section(Some("test_helper")) == Status::Ok {
-        ctx.add_info_field_str("field", "test_helper_value");
-    }
+#[derive(Debug, Clone, InfoSection)]
+struct InfoData {
+    field: String,
+    dictionary: HashMap<String, String>,
+}
+
+fn add_info(ctx: &InfoContext, _for_crash_report: bool) -> RedisResult<()> {
+    let mut dictionary = HashMap::new();
```

Comment:
Can we keep the exact old behaviour here?

Reply:
I agree but we should keep it just to make sure it works, maybe even add a deprecation notice on this example?

Reply:
Sounds good, I only care to keep some sanity test that make sure the old API still works. Even if we are sure it is not broken now, it will keep us from breaking it in the future until we will drop it...

Reply:
Sorry for been pedantic but I think its very important. It is still not exactly the same, the return type is changed:
https://github.com/RedisLabsModules/redismodule-rs/pull/355/files#diff-ccfc40870ea0b55ee48ea454932642f33976a4c616ab31982d299902114a28c8R35